README.md

```diff
@@ -6,16 +6,19 @@ Currently, this bundle is designed to work only with a Shopify store, but could
 
 ### Installation
 
-1. Run `composer require torqit/store-syndicator-bundle` in your project's terminal. 
-2. In the same terminal, run `bin/console pimcore:bundle:enable StoreSyndicatorBundle` 
+1. Run `composer require torqit/store-syndicator-bundle` in your project's terminal.
+2. In the same terminal, run `bin/console pimcore:bundle:enable StoreSyndicatorBundle`
 3. Then run `bin/console pimcore:bundle:install StoreSyndicatorBundle`
 
 ### Setting up an export:
 
 1. Create a new "TorqStoreExporterShopifyCredentials" Data Object with your Shopify store's credentials.
 2. Create a new "Store Exporter" item in the Pimcore Datahub.
 3. In your created Store Exporter, under the API Access tab, drag in your credentials object created in step 1.
-4. Under the Choose Products tab, choose the Data Object class you will be exporting, and one or multiple folders or objects you wish to export.
+4. Under the Choose Products tab, choose the Data Object class you will be exporting, and use the "To Export" and "Don't Export" sections to choose your products like follows:
+   - Fill the "To Export" section with one or multiple folders or objects you wish to export.
+   - Fill the "Don't Export" section with any exceptions in the "To Export" section
+   - This way it is possible to select your entire products folder and filter out any number of troublesome objects or folders
```

Comment:
Comma/space-separated? And do you need to provide IDs?

Reply:
I added an image to the readme (i dont know why its not showing up in the commit history). But the system is drag and drop.